ocp4/profiles/coreos-ncp.profile

```diff
@@ -579,3 +576,6 @@ selections:
     # AU-11
     - auditd_data_retention_num_logs
     - auditd_data_retention_max_log_file
+
+    # AC-2(5), AC-12
+    - accounts_tmout
```

Comment:
Is a variable also needed?

Reply:
@shawndwells it should use the default variable assigned which is 10 minutes.